src/applications/mhv-medical-records/containers/VitalDetails.jsx

```diff
@@ -122,11 +122,19 @@ const VitalDetails = props => {
   useEffect(
     () => {
       if (records?.length) {
-        focusElement(document.querySelector('h2'));
         paginatedVitals.current = paginateData(records);
         setCurrentVitals(paginatedVitals.current[currentPage - 1]);
       }
     },
+    [records],
+  );
+
+  useEffect(
+    () => {
+      if (currentPage > 1 && records?.length) {
+        focusElement(document.querySelector('h2'));
```

Comment:
Can we make this use an `id` like the RecordList version? A page can have more than one `<h2>` and if we ever add another one here, this might break.

Reply:
Added id to h2 element.